hack/make/dyngccgo

```diff
@@ -15,7 +16,7 @@ fi
 
 (
 	export IAMSTATIC="false"
-	export EXTLDFLAGS_STATIC_DOCKER=''
+	export EXTLDFLAGS_STATIC_DOCKER='-static-libgcc -static-libgo -lnetgo'
```

Comment:
does this mean that even when building dynamic-linked docker, we are going to use the netgo DNS routines?